test/parallel/test-parse-args.mjs

```diff
@@ -0,0 +1,572 @@
+import '../common/index.mjs';
+import assert from 'assert';
+import { test } from 'node:test';
+import { parseArgs } from 'util';
```

Comment:
If we're using `node:test`, we might as well use the `node:` prefix for all core modules imports (non-blocking suggestion, just a thought)
```suggestion
import assert from 'node:assert';
import { test } from 'node:test';
import { parseArgs } from 'node:util';
```